profile-management/OktaProfileManagement/ProfileManagement/UserProfile.swift

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2019 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+struct User : Codable {
+    struct Profile: Codable {
```

Comment:
style-nit: The `User` struct has a colon w/ a space before, but the `Profile` struct does not. Lets make sure we're consistent